### DISCUSSION TITLE
Add the option to remove vertex snapping

### DISCUSSION THREAD

**Discussion by Zakawer** [117HD/RLHD#500] [Ideas]
The other RuneLite GPU plugin already has a "remove vertex snapping" option. Turning on this option makes most in-game animations look a lot more fluid and makes in-game models look better overall, especially if animation smoothing (tweening) is disabled.

I think this function should be added to 117 HD.

**Comment by SirFancyBacon** [comment]
Well good news! we also thought the option should be added! :)
Right now i believe it is forced on so you are currently seeing the benefits of it; Next release will have a toggle for it.
![image](https://github.com/user-attachments/assets/f77cca02-8219-412a-a838-6197080761cd)
